civitai-downloader-v2/src/cli/main.py

```
@@ -0,0 +1,424 @@
+                click.echo("✅ File appears safe")
+                click.echo(f"   Risk level: {scan_result.scan_result.value}")
+            else:
+                click.echo("⚠️  Security threats detected!")
+                click.echo(f"   Risk level: {scan_result.scan_result.value}")
+                click.echo(f"   Issues found: {len(scan_result.issues)}")
+                
+                for issue in scan_result.critical_issues + scan_result.high_issues:
+                    click.echo(f"   - {issue.severity.upper()}: {issue.description}")
+            
+            if detailed:
+                click.echo(f"\n📊 Detailed Results:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)